build/buildall

```diff
@@ -118,9 +118,12 @@ done
 DIST_PROFILE=${DIST_PROFILE:-"noSnapshots"}
 [[ "$MODULE" != "" ]] && MODULE_OPT="--projects $MODULE --also-make" || MODULE_OPT=""
 
+with_db_build=""
+
 case $DIST_PROFILE in
 
   snapshots?(WithDatabricks))
+    with_db_build="true"
```

Comment:
the WithDatabricks is optional here, its not always true.  if I just use "snapshots" then its not with databricks.  You need to check explicitly for WithDatabricks before setting this.